src/main/clojure/clara/macros.clj

```diff
@@ -131,7 +131,8 @@
           :test
           `(eng/->TestNode
             ~id
-            ~(com/compile-test id (:constraints condition))
+            ~(:env beta-node)
+            ~(com/compile-test id (:constraints condition) (:env beta-node))
```

Comment:
I think this would have fixed the ClojureScript compiler error I see in the build that failed? https://github.com/cerner/clara-rules/actions/runs/5136943790/jobs/9244352594

Reply:
Yes, I had missed this in my initial pass.